mission-control-client/src/components/dashboard/admin-dashboard/DashboardMetrics.js

```diff
@@ -1,9 +1,9 @@
-import React from "react";
+import React, {useEffect} from "react";
```

Comment:
What is this change for?

Reply:
It was used for a console log while building out the functionality for the new feature, it can be disregarded.